mock-caas/src/test/java/com/sequenceiq/caas/grpc/service/MockUserManagementServiceTest.java

```diff
@@ -61,4 +65,44 @@ public void testCreateWorkloadUsername() {
 
         Assert.assertEquals(expected, underTest.sanitizeWorkloadUsername(username));
     }
+
+    @Test
+    public void testGetActorWorkloadCredentials() {
+        GetActorWorkloadCredentialsRequest req = GetActorWorkloadCredentialsRequest.getDefaultInstance();
+        MockUMSServiceCallStreamObserverImpl<GetActorWorkloadCredentialsRequest, GetActorWorkloadCredentialsResponse> responseObserver
+            = new MockUMSServiceCallStreamObserverImpl();
+
+        underTest.getActorWorkloadCredentials(req, responseObserver);
+        Assert.assertNotNull("Response must not be null", responseObserver.getResponse());
```

Comment:
I think this mocking could be done more idiomatically using mockito like:

```
StreamObserver s = Mockito.mock(StreamObserver.class);
underTest.getActorWorkloadCredentials(req, s);
Mockito.verify(s).onNext(Matchers.notNull());
Mockito.verify(s).onCompleted();
Mockito.verify(s, Mockito.never()).onError(Mockito.any());
```

Reply:
(but I still think you could just remove this test entirely since it doesn't really do anything past duplicating the implementation)

Reply:
I agree that this test isn't necessary

Reply:
Removed test.